packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,19 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private updateTitle(){
+        // Update the page <title> when the hash changes
+        let title:string;
```

Comment:
Rather than having a mutable `title` variable, consider pulling this logic out into a `getTitle()` function, and returning immediately once you've determined the appropriate value.